src/commands/hosting-channel-deploy.ts

```diff
@@ -102,9 +109,28 @@ export default new Command("hosting:channel:deploy [channelId]")
                 logger.debug("[hosting] updated TTL for existing channel for site", site, chan);
               }
             }
+            try {
+              await syncAuthState(projectId, site);
+            } catch (e) {
+              // not sure if we actually want to print a warning when we cant sync?
+              logLabeledWarning(LOG_TAG, "Unable to sync Firebase Auth state.");
+            }
```

Comment:
Could this syncAuthState be called after both the "channel exists" and "channel doesn't exist" logic is done? Sure it means that we might list the channels more often than necessary, but it would reduce the number of locations where we deal with auth state - I think that's a worthy tradeoff.

Reply:
I am fine with that just thought that the design specifies to syncAuthState only after a deploy to an existing channel